en/202407/exploring-geo-distribution-in-emqx-for-enhanced-scalability.md

```diff
@@ -0,0 +1,258 @@
+
+Strictly speaking, when we are talking about latency, we need to take into account *throughput* as well. Even a high-latency network can have a sufficiently high throughput. However, the unreliable nature of the network usually makes it hard to achieve high throughput. In the case of TCP connections, even a single packet loss can cause a significant drop in throughput, because the TCP stack will have to retransmit the packet and significantly scale down the transmission window. Again, the further away communication peers are from each other, the more often this will happen.
+
+Adding to the problem, raw network throughput is usually not crucial for many clients. Rather, the throughput of various *operations* conducted over the [MQTT protocol](https://www.emqx.com/en/blog/the-easiest-guide-to-getting-started-with-mqtt) is what counts. For example, throughput of message publishing, or throughput of `SUBSCRIBE` operations. It is where the *shared state* comes into play and, more importantly, the need to *coordinate* updates of this state.
+
+Thanks to the nature of the MQTT protocol, which operates in an asynchronous manner and where the clients are relatively independent of each other, the insidiousness of the shared state is not that prominent. Yet, it is still there. But first, to better understand the problem, let's start with setting up a basic geo-distributed EMQX cluster.
+
+## A Cluster 12600 km Wide
+
+As you might already know, EMQX 5 follows a new deployment model, where the cluster consists of two types of nodes: *core* and *replicant*. This model was specifically designed to support a broader range of deployment scenarios, including geo-distribution, though with some pitfalls, which we will explore later. Under this model, a cluster consists of 2 types of nodes: core nodes that manage essential parts of the shared state and replicant nodes that do not participate in the state management but simply replicate state changes made by the core nodes.
+
+Nevertheless, let us first deploy a cluster in a more traditional fashion, so we can better understand where the benefits of the new model could come from. Our “traditional” cluster will consist of 3 core nodes geographically distributed over 3 continents.
+
+| **Location**             | **Node name**        |
+| :----------------------- | :------------------- |
+| Frankfurt (eu-central-1) | `emqx@euc1.emqx.dev` |
+| Hong Kong (ap-east-1)    | `emqx@ape1.emqx.dev` |
+| Cape Town (af-south-1)   | `emqx@afs1.emqx.dev` |
+
+![EMQX Geo-Distribution](https://assets.emqx.com/images/460cdbd66a7e51e84f988f1574874025.png)
```

Comment:
This map actually illustrates the cluster that is subject of "Replicants to the Rescue" section, I believe it should be moved here. This deployment should have "triangular" shape, basically joining 3 geo-points mentioned above.